src/library/Ora/ReadModel/Task.php

```diff
@@ -32,6 +36,7 @@ class Task extends EditableEntity
     
     CONST ROLE_MEMBER = 'member';
     CONST ROLE_OWNER  = 'owner';
+    CONST NOT_MEMBER = 'notmember';
```

Comment:
La creazione di un valore diverso da null ha senso solo se null non è semanticamente applicabile alla funzione. Se la funzione rappresenta una domanda, ovvero in questo caso: qual è il ruolo di questo membro? Hai 2 alternative: 1 ritorni un'eccezione se l'utente non è membro, perché ritieni sbagliata una premessa alla domanda, oppure, in modo meno rigido, rispondi "Nessuno" e null rappresenta esattamente l'assenza di informazione, per cui è semanticamente pertinente. Non ti serve un valore ad hoc.